tests/contrib/minikube/test_kubernetes_pod_operator.py

```diff
@@ -37,7 +39,7 @@ def test_working_pod(self):
             namespace='default',
             image="ubuntu:16.04",
             cmds=["bash", "-cx"],
-            arguments=["echo", "10"],
+            arguments=["echo 10"],
```

Comment:
Just curious, any specific reason why you've changed this?

Reply:
Sure! I was getting weird errors when I attempted to perform other commands using the comma separation (especially if I tried to do multiple commands with `&&`). since `bash -cx` takes in a string anyways, I figured this would be a better way to approach it for if anyone was reading the tests for examples.